examples/chain/44_DriftSolver/example.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strings"
+
+	_ "embed"
+
+	"github.com/FluxNFTLabs/sdk-go/chain/modules/astromesh/types"
+	strategytypes "github.com/FluxNFTLabs/sdk-go/chain/modules/strategy/types"
+	chaintypes "github.com/FluxNFTLabs/sdk-go/chain/types"
+	chainclient "github.com/FluxNFTLabs/sdk-go/client/chain"
+	"github.com/FluxNFTLabs/sdk-go/client/common"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	//go:embed drift_solver.wasm
+	intentSolverBinary []byte
+)
+
+
+func main() {
+	networkName := "local"
+	if len(os.Args) > 1 {
+		networkName = os.Args[1]
+	}
+	network := common.LoadNetwork(networkName, "")
+	kr, err := keyring.New(
+		"fluxd",
+		"file",
+		os.Getenv("HOME")+"/.fluxd",
+		strings.NewReader("12345678"),
+		chainclient.GetCryptoCodec(),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	// init grpc connection
+	cc, err := grpc.Dial(network.ChainGrpcEndpoint, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		panic(err)
+	}
+
+	// init client ctx
+	clientCtx, senderAddress, err := chaintypes.NewClientContext(
+		network.ChainId,
+		"user1",
+		kr,
+	)
+	if err != nil {
+		panic(err)
+	}
+	clientCtx = clientCtx.WithGRPCClient(cc)
+
+	// init chain client
+	chainClient, err := chainclient.NewChainClient(
+		clientCtx,
+		common.OptionGasPrices("500000000lux"),
+	)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	msg := &strategytypes.MsgConfigStrategy{
+		Sender:   senderAddress.String(),
+		Config:   strategytypes.Config_deploy,
+		Id:       "",
+		Strategy: intentSolverBinary,
+		Query:    &types.FISQueryRequest{},
+		Metadata: &strategytypes.StrategyMetadata{
+			Name:        "Nexus Transfer Solver",
+			Description: "Simplifies financial transfers by allowing retail users to batch multiple requests using easy, human-readable prompts.",
```

Comment:
Description: `
Solver simplifies drift interaction. Available options: 
leverage: 1..20,
market: 'btc-usdt', 'eth-usdt', 'sol-usdt',
auction_duration: 10..255,
percent: 1..100,
taker_svm_address: base58 pubkey of taker
`